third_party/terraform/utils/config_test.go

```diff
@@ -130,7 +130,7 @@ func TestAccConfigLoadValidate_accessTokenImpersonated(t *testing.T) {
 	proj := getTestProjectFromEnv()
 	serviceaccount := multiEnvSearch([]string{"IMPERSONATE_SERVICE_ACCOUNT_ACCTEST"})
 
-	c, err := google.CredentialsFromJSON(context.Background(), []byte(creds), testOauthScope)
+	c, err := google.CredentialsFromJSON(context.Background(), []byte(creds), DefaultClientScopes..)
```

Comment:
I think we want testOauthScope appended to DefaultClientScopes, if that makes sense? Sorry for the confusion!

Reply:
Just DefaultClientScopes is fine! See https://github.com/GoogleCloudPlatform/magic-modules/pull/4015#discussion_r505749384

Reply:
It is already done.

Reply:
oops!  sorry about that - it looks like there's a `.` missing after `DefaultClientScopes` here

Reply:
😬  fixed